### PR TITLE
Add coverage test for udevadm

### PR DESCRIPTION
**Pull request by dtardon**

### CODE REVIEW

**Review by yuwata — APPROVED** [review]

File: test/units/testsuite-17.10.sh [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+udevadm trigger --name-match /sys/class/net/$netdev
+udevadm trigger --name-match /sys/class/net/$netdev --name-match /dev/null
+udevadm trigger -b /sys/class/net/$netdev
+udevadm trigger --parent-match /sys/class/net/$netdev --name-match /dev/null
+udevadm trigger --initialized-match
+udevadm trigger --initialized-nomatch
+udevadm trigger -w
+udevadm trigger --uuid /sys/class/net/$netdev
+udevadm trigger --wait-daemon
+udevadm trigger --wait-daemon=1
```
Comment (yuwata) [inline test/units/testsuite-17.10.sh]: Maybe, the timeout is too short for slow CI environment?

Comment (dtardon) [reply test/units/testsuite-17.10.sh]: Maybe. I'll increase it to 5; hopefully that'd be enough. (And the other timeouts too, just to be sure. Coverage tests are mainly useful when run with sanitizers, which always slow things down.)

**Comment by yuwata** [comment]
The CI failures are unrelated, caused by my recent PR.

**Pull request merged**